store/tikv/region_request.go

```diff
@@ -194,7 +194,7 @@ func (s *RegionRequestSender) onRegionError(bo *Backoffer, ctx *RPCContext, regi
 		if notLeader.GetLeader() != nil {
 			boType = BoUpdateLeader
 		} else {
-			boType = BoRegionMiss
+			boType = boLeaderNotReady
```

Comment:
Use `HandleRegionError`?

Reply:
No. `notLeader` will be retried inside func `SendReq`, but `HandleRegionError` is used when the request may not belong to the region.

Reply:
Maybe the name `HandleRegionError` is not precise.

Reply:
I think `HandleRegionErr` has nothing to do with whether the request is still belong to the region. We can handle `boUpdateLeader` and `boLeaderNotReady` in `HandleRegionErr` too.